dev/com.ibm.ws.microprofile.openapi_fat/.classpath

```diff
@@ -2,7 +2,7 @@
 <classpath>
 	<classpathentry kind="src" path="fat/src"/>
 	<classpathentry kind="src" path="test-applications/appWithAnnotations/src"/>
-	<classpathentry kind="src" path="test-applications/appWithStaticDoc/src"/>
+	<classpathentry kind="src" path="test-applications/appWithStaticDoc/resources"/>
```

Comment:
Since there is no java source in this directory, I didn't think we included it in .classpath.  At least other projects that have 'resources' directories with just XML have not done this.